homeassistant/components/opower/coordinator.py

```diff
@@ -245,6 +329,31 @@ def _update_with_finer_cost_reads(
                         break
             cost_reads += finer_cost_reads
 
+        def _check_is_cost_missing(
```

Comment:
Can this just be:

```python
        def _check_is_cost_missing(cost_reads: list[CostRead]) -> bool:
            if not cost_reads:
                return False
            return sum(cost_read.provided_cost for cost_read in cost_reads) == 0
```

And only call it with daily_cost_reads and hourly_cost_reads?

You will also set: `cost_reads_billing = cost_reads.copy()` right after we fetch bill data.